src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -58,6 +58,7 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
   let (bindings, validationErrors) =
     log.LogTrace("[{BindingNameChain}] Initializing bindings", nameChain)
     let bindingDict = Dictionary<string, VmBinding<'model, 'msg, obj>>(bindings.Length)
+    let validationDict = Dictionary<string, string list ref>()
```

Comment:
Why move this here?

Reply:
It needed to move up, and it seemed natural to put it there since it's right beside the other dictionary we are populating in the below code.

Reply:
I don't see why it needed to move up.  Why did it need to move up?

Reply:
On line 86 I'm adding to the dictionary, which was previously instantiated after line 88.

Reply:
Ok. I understand now. Thanks.